modules/models/course_instructor.php

```diff
@@ -0,0 +1,22 @@
+<?php require_once (dirname(__FILE__)."/record.php");
+
+class CourseInstructor extends Record {
+
+    static protected $table_name = "course_instructors";
+    
+    protected static $belongs_to = array(
+        "user" => array(
+            "class_name" => "User",
+            "foreign_key" => "user_id",
+        ),
+        "courses" => array(
+            "class_name" => "Courses",
+            "foreign_key" => "course_id",
+        )
+    );
+
+    public $course_id;
+    public $user_id;
+	public $created_at;
```

Comment:
seems to be extra indentations here

Reply:
good catch

Reply:
hmm its not here on my end

Reply:
you have tabs instead of whit space or maybe its the other way around

Reply:
I tried removing it, it doesn't appear on my end, not sure what to do to be honest...